### ISSUE TITLE
Exception caught: Could not find file(s)

### ISSUE THREAD

**Issue by peddanet** [sim0n00ps/OF-DL#606]
System: Ubuntu 22.04, .Net8, OFDL built v.1.7.79:

When downloading posts it continously tells me at some users:

```
Downloading 2265 Posts ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   3%
                                                                    Exception caught: Could not find file '/home/sss/Desktop/ssss/fuck/of/sss/Posts/Free/Videos/0hteq4o3mzyjoa1cidw51_source.mp4'.

StackTrace:    at System.IO.FileInfo.get_Length()
   at OF_DL.Helpers.DownloadHelper.DownloadDrmMedia(String user_agent, String policy, String signature, String kvp, String sess, String url, String decryptionKey, String folder, Dat                                                                    sk, String customFileName, String filename, String path) in /home/xxx/Projects/OF-DL-OFDLV1.7.79/OF DL/Helpe
```
STacktrace unfortunately not fully displayed in console.

Is this known? I mean the directory itself is accessible (at least for me, and at least with v1.7.60 it worked).....

**Comment by melithine** [comment]
What version of ffmpeg are you using? I recommend at least 7.0.2.

**Comment by peddanet** [comment]
Well it seems , Ubuntu 22.04 came with ffmpeg version 6.1.1! So I will try updating! Thanks melithine!

**Comment by peddanet** [comment]
Ok it was a little bit complicated. I forgot I have snap versions installed, and two self built versions, which needed first  to be removed. 

Then I followed this guide: [Ubuntu Handbook: Install ffmpeg 7](https://ubuntuhandbook.org/index.php/2024/04/ffmpeg-7-0-ppa-ubuntu/)

I restarted OF DL and so far it went error free through! If errors still persist, I will try to reopen this one or reissue a new one, thanks for your help , @melithine !
